plugins/nginx-vhosts/functions

```diff
@@ -22,6 +22,34 @@ fn-nginx-access-log-path() {
   fn-plugin-property-get-default "nginx" "$APP" "access-log-path" "${NGINX_LOG_ROOT}/${APP}-access.log"
 }
 
+fn-nginx-proxy-buffer-size() {
+  declare desc="get the configured proxy buffer size"
+  declare APP="$1"
+
+  fn-plugin-property-get-default "nginx" "$APP" "proxy-buffer-size" "$(getconf PAGESIZE)"
```

Comment:
Where is `getconf` from?

Reply:
It's from `$PATH`. Probably I should have a function like `fn-nginx-vhosts-nginx-location` to get `getconf` location instead of directly invoking it?

Reply:
Is that from a specific OS package? If so, we'll want to have that as a dependency.

Reply:
Hmmm feeling like introducing a dependency is not a good idea although `getconf` should be available on most distros. How about we just hide the property key-value pair in `nginx.conf` if the value is not set? Let Nginx use its default value.